pkgs/by-name/an/anki/addons/fsrs4anki-helper/default.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  anki-utils,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+
+anki-utils.buildAnkiAddon (finalAttrs: {
+  pname = "fsrs4anki-helper";
+  version = "24.06.3-unstable-2025-10-24";
+
+  src = fetchFromGitHub {
+    owner = "open-spaced-repetition";
+    repo = "fsrs4anki-helper";
+    rev = "2bd5fb0816885990e73a7e95bf07690c2e47c368";
+    hash = "sha256-Vu99QaMb++Umke6ANC83EGPQJSa3rnGCnhBVP/zm7V8=";
+    fetchSubmodules = true;
+  };
+
+  passthru.updateScript = unstableGitUpdater { tagPrefix = "v"; };
+
+  meta = {
+    description = "Anki add-on that supports the FSRS algorithm";
+    longDescription = ''
```

Comment:
Would be good to mention <https://github.com/open-spaced-repetition/fsrs4anki-helper/blob/main/config.md>.